circle.yml

```diff
@@ -82,7 +82,9 @@ jobs:
           name: Install Docker Compose
           command: |
             set -x
-            pip install docker-compose>=1.18
+            curl -L https://github.com/docker/compose/releases/download/1.19.0/docker-compose-`uname -s`-`uname -m` > ~/docker-compose
+            chmod +x ~/docker-compose
+            sudo mv ~/docker-compose /usr/local/bin/docker-compose
```

Comment:
Why are we doing this compared to installing via pip? Maybe add a comment?

Reply:
This is changed because I noticed it was different from my local config, so we now install docker-compose using curl, and a local tester should be able to use this script as well. 

For the script within ```scripts/ui-hostname-setup.sh``` we had a different config from the usual localenv. Well maybe it was the same, but I noticed on my mac, which has docker compose installed via Pip the names given to the container contained no spaces or ```-```'s.
```addonsserver_addons-frontend_1``` vs ```addons-server_addons-frontend_1```.